msgraph-connector-demo/pom.xml

```diff
@@ -23,13 +25,48 @@
       </snapshots>
     </pluginRepository>
   </pluginRepositories>
+
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <configuration>
+              <appendAssemblyId>false</appendAssemblyId>
+              <descriptors>
+                <descriptor>../maven/deploy/zip.xml</descriptor>
+              </descriptors>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
         <version>9.4.0-SNAPSHOT</version>
         <extensions>true</extensions>
+        <executions>
+          <execution>
+            <id>test.app.deploy</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>deploy-to-engine</goal>
+            </goals>
+            <configuration>
+              <deployMethod>HTTP</deployMethod>
```

Comment:
can we get a Jenkinsfile that will actually do the deployment to any host passed as parameter?